src/metabase/metabot.clj

```diff
@@ -104,26 +105,26 @@
   [& _]
   (let [cards (with-metabot-permissions
                 (filterv mi/can-read? (db/select [Card :id :name :dataset_query], {:order-by [[:id :desc]], :limit 20})))]
-    (str "Here's your " (count cards) " most recent cards:\n" (format-cards cards))))
+    (tru "Here's your {0} most recent cards:\n{1}" (count cards) (format-cards cards))))
 
 (defn- card-with-name [card-name]
   (first (u/prog1 (db/select [Card :id :name], :%lower.name [:like (str \% (str/lower-case card-name) \%)])
            (when (> (count <>) 1)
-             (throw (Exception. (str "Could you be a little more specific? I found these cards with names that matched:\n"
+             (throw (Exception. (tru "Could you be a little more specific? I found these cards with names that matched:\n{0}"
```

Comment:
`^String`

Reply:
Sorry, @camsaul, I don't know what you mean by this comment (or the other two places you've commented this in this file). Do you want me to change something here?

Reply:
Oh yeah I meant that for @senior. 

@senior did we decide type hinting with `^String` doesn't work?

Reply:
@mazameli I went ahead and fixed these.

So basically the `tru` macro doesn't tell people it's returning a string. And certain things like `(Exception.)` need to know they're getting a string to work properly. So we have to tell Clojure that the output of `tru` is a string since it's not doing it itself.

The easiest way I think is just to wrap that in a call to `(str)` which *does* advertise the fact that it returns a string.

So TL;DR do

```clojure
(Exception. (str (tru "...")))`
```

instead of

```clojure
(Exception. (tru "..."))
```

In almost all other places you don't need to wrap things in `(str)` so you can just continue to do those the normal way. If there's another place you need to do it the linter will catch it for you